### PR TITLE
Generate new Substrate account

### PR DESCRIPTION
**Pull request by EugenWay**
Features:

- Modal Popup
- Automatic generation of new account after it is opened
- mnemonic/seed camp for copy and save
- Save JSON 

@reviewer-or-team

### CODE REVIEW

File: website/frontend/src/helpers.ts [hunk 1]
```diff
@@ -28,3 +28,12 @@ export function readFileAsync(file: File){
     reader.readAsArrayBuffer(file);
   });
 };
+
+
+export default function toShortAddress(_address: string) {
```
Comment (codev0) [inline website/frontend/src/helpers.ts:33]: better way to use exports is named,
instead 
```
export default function toShortAddress
//
import toShortAddress from '../utils'
```
use
```
export function toShortAddress
//
import { toShortAddress } from '../utils'
```

File: website/frontend/src/components/blocks/Modal/index.tsx [hunk 2]
```diff
@@ -0,0 +1,3 @@
+import Modal from './Modal';
```
Comment (codev0) [inline website/frontend/src/components/blocks/Modal/index.tsx:1]: you should change it to named export

`export { Modal } from './Modal'`

File: website/frontend/src/components/blocks/Keyring/index.tsx [hunk 3]
```diff
@@ -0,0 +1,3 @@
+import Keyring from './Keyring';
```
Comment (codev0) [inline website/frontend/src/components/blocks/Keyring/index.tsx:1]: `export { Keyring } from './Keyring'`

**Review by codev0 — CHANGES_REQUESTED** [review]

File: website/frontend/src/components/blocks/Keyring/Keyring.tsx [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+      setKey(isSeed);
+    }
+
+    if (event.target.value === 'mnemonic') {
+      setKey(isMnemonic);
+    }
+  };
+
+  const handleCreate = () => {
+    localStorage.setItem('gear_mnemonic', JSON.stringify(isJson));
```
Comment (osipov-mit) [inline website/frontend/src/components/blocks/Keyring/Keyring.tsx:68]: It's better to rename the variable to keyring_json or keyPairJson

Should be checked where it is used

**Review by osipov-mit — CHANGES_REQUESTED** [review]

File: website/frontend/src/helpers.ts (hunk 1 above)
Comment (EugenWay) [reply website/frontend/src/helpers.ts]: done

File: website/frontend/src/components/blocks/Keyring/Keyring.tsx (hunk 4 above)
Comment (EugenWay) [reply website/frontend/src/components/blocks/Keyring/Keyring.tsx]: I agree.  isJson -> keyPairJson

**Review by codev0 — APPROVED** [review]
Good job

**Review by osipov-mit — APPROVED** [review]

**Pull request merged**